<?xml version='1.0' encoding='UTF-8'?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" version="2.0">
  <channel>
    <title>CVE Feed for DataDog -- guarddog</title>
    <link>https://raw.githubusercontent.com/deepseas/cvelistV5monitor/main/feeds/datadog/guarddog.rss</link>
    <description>The latest CVEs for DataDog -- guarddog</description>
    <docs>http://www.rssboard.org/rss-specification</docs>
    <generator>python-feedgen</generator>
    <lastBuildDate>Sat, 03 Aug 2024 20:22:07 +0000</lastBuildDate>
    <ttl>60</ttl>
    <item>
      <title>CVE-2022-23530|2024-08-03T03:43:46.472Z -- datadog -- guarddog
</title>
      <link>https://www.cve.org/CVERecord?id=CVE-2022-23530</link>
      <description>GuardDog is a CLI tool to identify malicious PyPI packages. Versions prior to v0.1.8 are vulnerable to arbitrary file write when scanning a specially-crafted remote PyPI package. Extracting files using shutil.unpack_archive() from a potentially malicious tarball without validating that the destination file path is within the intended destination directory can cause files outside the destination directory to be overwritten.  This issue is patched in version 0.1.8. Potential workarounds include using a safer module, like zipfile, and validating the location of the extracted files and discarding those with malicious paths.</description>
      <guid isPermaLink="false">CVE-2022-23530|2024-08-03T03:43:46.472Z</guid>
      <pubDate>Fri, 16 Dec 2022 22:56:33 +0000</pubDate>
    </item>
    <item>
      <title>CVE-2022-23531|2024-08-03T03:43:46.507Z -- datadog -- guarddog
</title>
      <link>https://www.cve.org/CVERecord?id=CVE-2022-23531</link>
      <description>GuardDog is a CLI tool to identify malicious PyPI packages. Versions prior to 0.1.5 are vulnerable to Relative Path Traversal when scanning a specially-crafted local PyPI package. Running GuardDog against a specially-crafted package can allow an attacker to write an arbitrary file on the machine where GuardDog is executed due to a path traversal vulnerability when extracting the .tar.gz file of the package being scanned, which exists by design in the tarfile.TarFile.extractall function. This issue is patched in version 0.1.5.</description>
      <guid isPermaLink="false">CVE-2022-23531|2024-08-03T03:43:46.507Z</guid>
      <pubDate>Fri, 16 Dec 2022 23:41:15 +0000</pubDate>
    </item>
  </channel>
</rss>
